web/docs/about-vast/README.md

```diff
@@ -0,0 +1,228 @@
+  bulk, of which the majority are true negatives, (2) regular expression search
+  for finding patterns in command line invocations, URLs, or opaque string
+  messages, and (3) group-by and aggregations when hunting for threats or when
+  performing threshold-based detections. Data warehouses work well for (3) but
+  rarely for (1) and (2) as well.
+
+Bottom line: data warehouses may be well-suited for raw data processing, but
+a data backbone for security operations has a lot more domain-specific
+demands. The required heavy lifting to bridge this gap is cost and time
+prohibitive for any security operations center. This is why we built VAST.
+
+## Why not VAST?
+
+VAST is the right choice when you need to analyze large amounts of security
+telemetry for specific use cases in security operations. There exist a lot of
+database systems out there, and you may ask yourself the question: why not use
+that other system instead?
+
+To help answer this question, we offer a brief guidance when other systems might
+be a better fit.
+
+### Document DBs
+
+Document DBs, such as MongoDB, offer worry-free ingestion of unstructured
+data. They scale well horizontally and flexible querying.
+
+However, they might not be the best choice for the data plane in security
+operations, for the following reasons:
+
+- **Vertical Scaling**: when co-locating a storage engine next to high-volume
+  data sources, e.g., on a network appliance together with a network monitor,
+  CPU and memory constraints coupled with a non-negligible IPS overhead prohibit
+  scaling horizontally to build a "cluster in a box."
+
+- **Analytical Workloads**: the document-oriented storage does not perform well
+  for analytical workloads, such as group-by and aggregation queries. But such
+  analytics are very common in interactive threat hunting scenarios and in
+  various threshold-based detections. VAST leverages Arrow for columnar data
+  representation and partially for query execution.
+
+- **Economy of Representation**: security telemetry data exhibits a lot of
+  repetitiveness between events, such as similar IP addresses, URL prefixes, or
+  log message formats. This data compresses much better when transposed into a
+  columnar format, such as Parquet.
+
+A special case of document DBs are full-text search engines, such as
+ElasticSearch or Solr. The unit of input is typically unstructured text. The
+search engine uses (inverted) indexes and ranking methods to return the most
+relevant results for a given combination of search terms.
+
+:::note Recommendation
+However, most of the security telemetry arrives as structured log/event data, as
+opposed to unstructured textual data. If your primary use case involves working
+with text, VAST might not be a good fit. That said, it's still possible to
+leverage information retrieval techniques for security analytics. For example,
+Elastic is a popular SIEM that works well for medium-sized workloads but
+exhibits poor vertical scaling.
```

Comment:
It would really be great if we could cite benchmarks.

I found

https://elasticsearch-benchmarks.elastic.co/index.html#tracks/security/nightly/default/30d

Which is specifically aimed at security (last menu point on the left). Since they cite < 30k indexed documents on a 3-node-system, it would probably be easy to come up with a higher number (but of course we'd need a comparable data set).

Reply:
Is the raw data available somewhere? This would make for a really good comparison docs page.

Reply:
I might remove the relative comparison until we have benchmarks, otherwise we have no leg to stand on.